daisy/workflow.go

```diff
@@ -377,11 +377,25 @@ func (w *Workflow) getSourceGCSAPIPath(s string) string {
 }
 
 // PopulateClients populates the compute and storage clients for the workflow.
-func (w *Workflow) PopulateClients(ctx context.Context) error {
+func (w *Workflow) PopulateClients(ctx context.Context, options []option.ClientOption) error {
```

Comment:
You might consider `PopulateClients(ctx context.Context, options ... option.ClientOption)` The type of options is still []option.ClientOption:

https://yourbasic.org/golang/three-dots-ellipsis/